TigGUI/tigger

```diff
@@ -91,7 +92,7 @@ if __name__ == "__main__":
     import TigGUI.MainWindow
     dprint(1,"imported TigGUI.MainWindow")
     import TigGUI.Tools
-    from TigGUI.Tools import export_karma,add_brick,make_brick,restore_image
+
```

Comment:
As far as I remember, these things dynamically insert themselves into the menus when imported. So you may have just killed them here...

Reply:
i have no words for this. i'll restore it.

Reply:
Wait i see no difference in tigger If i comment it or uncomment it... what should this do?